discord/ext/pages/pagination.py

```diff
@@ -309,6 +319,8 @@ async def update(
             Whether the buttons get disabled when the paginator view times out.
         use_default_buttons: :class:`bool`
             Whether to use the default buttons (i.e. ``first``, ``prev``, ``page_indicator``, ``next``, ``last``)
+        default_button_row: :class:`int`
```

Comment:
This one seems to default to None
```suggestion
        default_button_row: Optional[:class:`int`]
```